nbbuild/antsrc/org/netbeans/nbbuild/MakeNBM.java

```diff
@@ -334,14 +334,14 @@ public void setProductDir( File dir ) {
     public void setFile(File file) {
         this.file = file;
     }
-
-    public void setUsePack200(boolean usePack200) {
+    //Commented below code, as pack200 is not supported from jdk 14 onwards
```

Comment:
I guess the methods (and so their attributes in the build.xml) should stay here, for compatibility. They, of course, won't do anything.

Reply:
Indeed, i'll address this comment.

Reply:
Addressed code review comments and removed all the previously commented code.